docs/flutter/v2/features/recording.mdx

```diff
@@ -10,25 +10,31 @@ Turn on RTMP Streaming or Recording!
 In 100ms, recording and streaming is usually achieved by having a bot join your room and stream what it sees and hears to a file (recording) or to an rtmp injest url (streaming).
 
 The topics covered in this doc are:
+
 1. [How to start streaming / recording.](#starting-streaming-recording)
 2. [How to stop streaming / recording.](#stopping-streaming-recording)
 3. [How to check the current status for streaming / recording.](#current-room-status)
 4. [When to check the current status](#when-to-check-for-room-status)
 
 ## Starting Streaming / Recording
+
 To start recording, streaming or both, create an instance of `HMSRecordingConfig`.
 
 `HMSRecordingConfig` takes the following:
-1. **meetingUrl**: *String*. The url the 100ms bot user will open to join your room. It must allow access without any user level interaction.
-2. **rtmpUrls**: *List<String\>*. If streaming is required, this has to be one or more RTMP Injest Urls with a max limit of 3 urls where the stream should go. If only recording, this can be an empty list.
-3. **record**: *Boolean*. If recording is required, set true. If recording is not required, set false. This value has no effect on streaming.
-4. **resolution**: *HMSResolution*. An optional value for the output resolution of the stream. For instance default is landscape at 1280x720 but this could be set for a portrait mode of 720x1280. Or smaller values like 480x80.
+
+1. **meetingUrl**: _String_. The url the 100ms bot user will open to join your room. It must allow access without any user level interaction.
+2. **rtmpUrls**: _List<String\>_. If streaming is required, this has to be one or more RTMP Injest Urls with a max limit of 3 urls where the stream should go. If only recording, this can be an empty list.
+    - Format: `rtmp://server.com/app/STREAM_KEY`
```

Comment:
this url makes it look like the `app` is a part of the expected input, while it's not. you can add a real example from youtube to make it look accurate